Dockerfile

```diff
@@ -41,7 +41,12 @@ RUN \
   apt-get -qq update && \
   apt-get -qq install apt-utils 2>/dev/null && \
   apt-get -qq dist-upgrade && \
+  echo "applying patch for https://github.com/docker-mailserver/docker-mailserver/issues/2023" && \
```

Comment:
This lacks context when referencing my issue as it's not a patch I detail.

Modify your comment referencing this PR about your workaround approach, or at the very least link to your comment in my referenced issue.

Reply:
> This lacks context when referencing my issue as it's not a patch I detail.

I used your code from https://github.com/docker-mailserver/docker-mailserver/issues/2023#issuecomment-855326403 -- what do you mean when you say it's not a patch I detail?

Reply:
My apologies, I wrote that ages ago and hadn't read through all my collapsed parts. Just saw that I suggested it as a viable workaround.

My issue was more of, how quickly can a maintainer find the relevant context from that issue and my verbose details. It should be easier for a maintainer to lookup the relevant context for the patch, so perhaps?:

```
Applying Dockerfile workaround detailed in: https://github.com/docker-mailserver/docker-mailserver/issues/2023#issuecomment-855326403
```